src/main/java/com/csv/communitytrackerjava/model/Project.java

```diff
@@ -29,4 +31,13 @@ public class Project extends BaseAuditClass {
     @Column(name="isactive", nullable = false)
     private Boolean isActive;
 
+    @OneToMany(mappedBy = "projectId", fetch = FetchType.LAZY)
+    private List<People> peoples;
+
+    public Project(Integer projectId, String projectDesc, String projectCode, Boolean isActive) {
```

Comment:
I think it's duplicate on AllArgsConstructor by Lombok?

Reply:
List of people is not included in this constructor

Reply:
Okay, I missed the people what is the class that using the constructor?